src/useReactiveState.ts

```diff
@@ -44,7 +44,7 @@ export const useReactiveState = <T>(
   if (
     prevDeps.current === undefined ||
     deps.length !== prevDeps.current.length ||
-    deps.some((d, i) => d !== prevDeps.current![i])
+    deps.some((d, i) => !Object.is(d, prevDeps.current![i]))
```

Comment:
I changed it because we do actually get NaN in the dependencies sometimes, now. (They come from the grid arrangement code when the width and height are not yet determined, giving aspect ratio = 0 / 0.)

Adding a comment.